search_resolution_translation: 0.075 # m 0.04
search_resolution_yaw: 0.3926991 # rad

use_full_object_point_cloud: false

perch_params:
  sensor_resolution_radius: 0.01 #m
  min_neighbor_points_for_valid_pose: 500 #50
  min_points_for_constraint_cloud: 50
  max_icp_iterations: 20
  icp_max_correspondence: 0.05 #m #0.075
  use_adaptive_resolution: false
  use_rcnn_heuristic: false
  use_model_specific_search_resolution: false

  ## Clutter mode
  use_clutter_mode: false
  # Should be in [0,1]
  clutter_regularizer: 0.0

  ## Visualization and Debugging
  visualize_expanded_states: true
  print_expanded_states: true
  debug_verbose: false # unused
